projects/igniteui-angular/src/lib/time-picker/themes/shared/_indigo.scss

```diff
@@ -0,0 +1,41 @@
+@use 'igniteui-theming/sass/bem' as *;
+@use 'igniteui-theming/sass/typography' as *;
+@use 'styles/themes/standalone' as *;
+@use '../light/tokens' as *;
+
+$_theme: $indigo;
+
+@include themed-block(igx-time-picker, indigo) {
+    @include e(header) {
+        border-block-end: rem(1px) solid var-get($_theme, 'divider-color');
+        gap: rem(2px);
+    }
+
+    @include e(header-hour) {
+        @include type-style(h5);
+    }
+
+    @include e(column) {
+        width: rem(54px);
+    }
+
+    @include e(item, $m: selected) {
+        @include type-style(h6);
+    }
+
+    @include e(buttons) {
+        padding: rem(8px) rem(16px);
+
+        [igxbutton] {
+            min-width: rem(88px);
+        }
+    }
+
+    @include m(vertical) {
+        @include e(header) {
+            min-width: rem(136px);
+            border-inline-end: rem(1px) solid var-get($_theme, 'divider-color');
+            border-block-end: none;
```

Comment:
If you add that border `border-block-end` only when the calendar is not vertical, you won't have to override it here; it looks cleaner in the end.